recipes/prgenv-gnu-openmpi/25.11/gh200/environments.yaml

```
@@ -0,0 +1,51 @@
+gcc-env:
+  compiler: [gcc]
+  network:
+      mpi: openmpi@5
+      specs:
+      - libfabric@2.3 +gdrcopy fabrics=lnx,shm,cxi # TODO: Add to/update alps-cluster-config
```

This should become simply

```suggestion
      mpi: openmpi
```
once https://github.com/eth-cscs/alps-cluster-config/pull/68 is finished and merged.